Mathlib/Probability/Notation.lean

```diff
@@ -6,7 +6,7 @@ Authors: Rémy Degenne
 import Mathlib.MeasureTheory.Function.ConditionalExpectation.Basic
 import Mathlib.MeasureTheory.Measure.Decomposition.Lebesgue
 
-/-! # Notations for probability theory
+/-! # Notation for probability theory
```

Comment:
I'd leave this one alone --- this is a different case. (Or you prefer to change it, please also adjust the module doc-string.)